shared-articles/what does dcdd do.md

```diff
@@ -0,0 +1,75 @@
+
+## dcrd responsibilities:
+dcrd is the backbone of Decred’s peer to peer network and has specific processes to conduct to keep the network functioning. Running a full node with dcrd contributes to the overall security of the network and increases the available paths for transactions and blocks to relay.
+
+* dcrd maintains the entire past transactional ledger of Decred
+* Allows relaying of transactions to other Decred nodes around the world
+* Maintains a transaction pool
+* Relays newly mined blocks
+* and finally ensures all transactions follow the rules
+
+In terms of benefits, since dcrd fully validates every block and transaction, it provides the highest security and privacy possible when used with a wallet that’s also in full validation mode. This is ideal for individuals, businesses, and services that need the most reliable and accurate data about transactions.
+
+## Node Hierarchy
+As Decred builds out its infrastructure, it becomes apparent that the underlying security is a high priority. Each time a new service is added, it increases the opportunity for node participation. This is done on multiple levels, and it’s important to note that not every node is equal.
+
+**A top tier fully validating node** is a dcrd unit run either as a stand-alone piece of software or as part of a service for 24 hours, 7 days a week, 365 days a year. This node has an inbound port open, so it can be publicly accessible for other nodes to connect to. This full node is the one that adds security to the chain and makes the project more decentralised. 
+
+**Side note: **There is also an option for running dcrd over the Tor network for greater privacy and security.
+
+**A second tier fully validating node** is a dcrd unit running intermittently or continuously, with the default settings which don’t include an open inbound port. This is not a publicly accessible node, which means other nodes in the network can’t connect to it, but it can connect to them. This full node is perfect for validating chain information and is good for personal privacy and security. This node setup doesn’t add to the security of the chain or make the project more decentralised.
+
+**Simple Payment Verification (SPV)** is not a node, but a wallet function. It allows the use of a Decred wallet without having to download the entire Decred blockchain. A wallet operating in SPV mode only needs to download full blocks containing transactions relevant to it and block headers. This reduces the wallet’s hardware requirements and greatly reduces the initial load time for new wallets. When an SPV wallet initialises, it will connect to the Decred network using dcrd peer connections.
```

Comment:
SPV doesn't really seem like it fits in this section, because, as you correctly say, it's not a full node. It's only "leeching" blocks that are relevant to it, and it's not sharing blocks back to the network.

Reply:
@PhoenixGreen What if you included a section which talked about the kinds of things that are NOT full nodes? That would help create a distinction for what sets `dcrd` apart from....things that aren't full nodes.

My understanding is limited here, so probably ask around in Matrix? However, I THINK the distinction is pretty much SPV. If it relies on SPV, then it's not a full node, because it's not downloading and validating every block.

It's kinda hard to list EVERY bit of software that might use SPV mode. `dcrwallet` can use SPV mode, OR it can use `dcrd`. In the same way, since `Decrediton` uses `dcrwallet` under the hood, the same is true of Decdrediton. It can EITHER rely on a full node, or it can rely on SPV mode.

So, to be honest, I don't really know what kinds of things are "Nodes, but not full nodes". And it would be super interesting to include that kind of info in this article.

Reply:
I would use a simple phrase like "public full nodes" to differentiate from other full nodes that are not publicly accessible (and directly helping the network).